bluebrain/deployment/environments/applications_libraries.yaml

```diff
@@ -9,6 +9,7 @@ spack:
           - gmsh
           - hdf5+mpi
           - highfive+mpi
+          - libsonata-report
```

Comment:
@jorblancoa : do we need to expose that as a module on BB5? i.e. do you expect our scientists to use it?

Its ok to install. Just wondering if it's going to be used via module. (Or some of us as a devs, just do `spack load`)

Reply:
makes sense, so maybe leave it only in specs?

Reply:
that's what I am thinking 👍